lib/ascii_binder/engine.rb

```diff
@@ -485,8 +481,8 @@ def configure_and_generate_page(topic,branch_config,navigation)
       if breadcrumb.length == 3
         subgroup_title = breadcrumb[1][:name]
         subgroup_id    = breadcrumb[1][:id]
-        dir_depth      = '../' + dir_depth
       end
+      dir_depth = '../' * topic.breadcrumb[-1][:id].split('::').length
```

Comment:
Same here. You need to catch the case where length = 0 to ensure that you don't end up with a nil value.